src/WrappedMToken.sol

```diff
@@ -680,9 +681,7 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
      * @return isApproved_ True if the account_ is a Registrar-approved earner, false otherwise.
      */
     function _isApprovedEarner(address account_) internal view returns (bool isApproved_) {
-        return
-            IRegistrarLike(registrar).get(_EARNERS_LIST_IGNORED) != bytes32(0) ||
-            IRegistrarLike(registrar).listContains(_EARNERS_LIST, account_);
+        return IEarnerStatusManager(earnerStatusManager).earnerStatusFor(account_);
```

Comment:
measure gas savings for inlining EarnerStatusManager logic within WrappedMToken vs calling external contract